app/src/main/res/layout/fragment_navigation_overlay_orig.xml

```diff
@@ -1,11 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<org.mozilla.tv.firefox.navigationoverlay.BrowserNavigationOverlayScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/overlayScrollView"
+
+<!-- TODO spacing is fixed in next commit, in order to make the diff more readable -->
+<LinearLayout
+    android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+<org.mozilla.tv.firefox.navigationoverlay.BrowserNavigationOverlayScrollView
+    android:id="@+id/overlayScrollView"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:layout_weight="1"
```

Comment:
Let's double check w/ UX about whether the hint bar should be overlapping or just underneath. (Or maybe this is more relevant to the overlay)

Reply:
It looks like...

> > […] on the homescreen, does all the content get moved up to accommodate the hint bar, or is it layered underneath and some part of the content would be hidden?
>
> The latter. The content is layered underneath the hint bar. The bar appears on top of some part of the content.

https://github.com/mozilla-mobile/firefox-tv/issues/1907#issuecomment-473718692